### PR TITLE
[Doc] The Bazel version is critical for compiling Ray

### PR DESCRIPTION
**Pull request by kevin85421**
## Checks

- [ ] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
    - [ ] I've added any new APIs to the API Reference. For example, if I added a 
           method in Tune, I've added it in `doc/source/tune/api/` under the 
           corresponding `.rst` file.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Comment by kevin85421** [comment]
cc @rickyyx the issue that we discussed yesterday.

File: doc/source/ray-contribute/development.rst [hunk 1]
```diff
@@ -363,6 +363,8 @@ run the following (via ``-c`` ``fastbuild``/``dbg``/``opt``, respectively):
 
 .. code-block:: shell
 
+ # Ray is sensitive to the Bazel version. Please ensure that you use the same
+ # version of Bazel as the one installed by ci/env/install-bazel.sh.
```
Comment (aslonnie) [inline doc/source/ray-contribute/development.rst:367]: people should install and use `bazelisk` instead.

we should also change all our `bazel` usage in docs into `bazelisk`, and then we do not need this comment.

**Comment by rickyyx** [comment]
should we call out at the bazel install section as well?

**Comment by aslonnie** [comment]
> should we call out at the bazel install section as well?

I (or @can-anyscale ) can take a look at this page, and give an update.

`ci/env/install-bazel.sh` today basically installs bazelisk and symlink bazel to bazelisk. it feels very CI specific, and should not really be recommended to devs on local machines today.

**Comment by kevin85421** [comment]
Thanks, @rickyyx @aslonnie! I will close this PR.

**Pull request closed without merging**